### PR TITLE
[Low Prio] Add a sqlite3 check

### PR DESCRIPTION
**Pull request by AlanCoding**
I got 3 failures running this locally, and ability to address those failures is the key thing I'd like to get out of this. I would like to position us to be multi-vendor regarding the database as much as we can reasonably do with minimal effort, and this is a nice start.

### CODE REVIEW

**Comment by AlanCoding** [comment]
Locally I'm easily able to replicate:

```
DJANGO_SETTINGS_MODULE=test_app.sqlite3settings py.test test_app/tests/ -k test_authenticator_map_validate
```

**Comment by AlanCoding** [comment]
Adding a link to the reason for the failure we're skipping https://github.com/django/django/pull/16601

File: test_app/tests/authentication/views/test_authenticator_map.py [hunk 1]
```diff
@@ -84,6 +85,9 @@ def test_authenticator_map_invalid_map_type(admin_api_client, local_authenticato
     assert '"invalid" is not a valid choice.' in response.data['map_type'][0]
 
 
+@pytest.mark.skipif(
+    connection.vendor == 'sqlite', reason='PositiveIntegerField min_value was not discovered but fixed in Django 5 https://github.com/django/django/pull/16601'
```
Comment (relrod) [inline test_app/tests/authentication/views/test_authenticator_map.py]: Can we also check `django.VERSION[0] < 5`, so this test works whenever we upgrade?

**Review by relrod — APPROVED** [review]

File: test_app/tests/authentication/views/test_authenticator_map.py (hunk 1 above)
Comment (AlanCoding) [reply test_app/tests/authentication/views/test_authenticator_map.py]: I was just being lazy. I'll push a commit.

**Pull request merged**